docs/source/getting-started/content/proc_basic-troubleshooting.adoc

```diff
@@ -48,3 +48,37 @@ If your issue has not been resolved by this procedure, perform the following ste
 . link:https://github.com/code-ready/crc/issues[Search open issues] for the issue that you are encountering.
 . If no existing issue addresses the encountered issue, link:https://github.com/code-ready/crc/issues/new[create an issue] and link:https://help.github.com/en/articles/file-attachments-on-issues-and-pull-requests[attach the [filename]`~/.crc/crc.log` file] to the created issue.
 The [filename]`~/.crc/crc.log` file contains detailed debugging and troubleshooting information which can help diagnose the problem that you are experiencing.
+
+= DNS configuration details
+
+This section describes how {prod} configures networking.
+This can be useful for troubleshooting, or in case you are using a non-standard DNS setup, and want to adjust it for {prod} needs.
+The [command]`{bin} ip` command can be used to obtain the VM IP address as needed.
+This address can change each time the VM is restarted.
+
+
+== Linux
+
+* {bin} expects NetworkManager to be used to manage networking.
+* NetworkManager is set up to use dnsmasq through the [filename]`/etc/NetworkManager/conf.d/crc-nm-dnsmasq.conf` configuration file.
+* the configuration file for this dnsmasq instance is [filename]`/etc/NetworkManager/dnsmasq.d/crc.conf`:
```

Comment:
Please capitalize the first word in each of these bullet points. They're complete sentences. :)

Reply:
Actually this part is obsolete after some commits today, I'll rework it.